modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -183,6 +183,15 @@ extern "C" {
 #endif
 #endif
 
+#ifndef USE_AV_SEND_FRAME_API
+// https://github.com/FFmpeg/FFmpeg/commit/7fc329e2dd6226dfecaa4a1d7adf353bf2773726
+#if LIBAVCODEC_BUILD >= CALC_FFMPEG_VERSION(57, 37, 100)
```

Comment:
@alalek Is this calculation correct?

Reply:
Please add check for "FFmpeg" (to avoid libav): `LIBAVCODEC_VERSION_MICRO >= 100`

Reply:
Fixed by https://github.com/opencv/opencv/pull/18039/commits/6494ff6038c19e41186404d3f12b56d4a34a319a

Reply:
I've squashed all commits :)